modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTxStateLocalMapTest.java

```diff
@@ -63,10 +64,13 @@ public class ItTxStateLocalMapTest extends IgniteAbstractTest {
 
     //TODO fsync can be turned on again after https://issues.apache.org/jira/browse/IGNITE-20195
     @InjectConfiguration("mock: { fsync: false }")
-    private static RaftConfiguration raftConfig;
+    private RaftConfiguration raftConfig;
```

Comment:
Any explanation of why it's not static anymore?

Reply:
There was a discussion among developers that static configurations might do more harm than good